src/reconcile/rec_write.c

```diff
@@ -2895,10 +2895,13 @@ __wt_bulk_init(WT_SESSION_IMPL *session, WT_CURSOR_BULK *cbulk)
 		WT_RET_MSG(session, EINVAL,
 		    "bulk-load is only possible for newly created trees");
 
-	/* Set a reference to the empty leaf page. */
+	/* Create an empty leaf page to insert into. */
 	pindex = WT_INTL_INDEX_COPY(btree->root.page);
 	cbulk->ref = pindex->index[0];
+	WT_RET(__wt_cache_read(session, cbulk->ref));
 	cbulk->leaf = cbulk->ref->page;
+	WT_RET(__wt_page_modify_init(session, cbulk->leaf));
+	__wt_page_only_modify_set(session, cbulk->leaf);
```

Comment:
The reason I used `__wt_cache_read` is so that the tree would exactly match what happens if an application goes to insert into that page.  For the same reason (the allocated page is hooked into a real tree), I didn't think any extra cleanup would be required.

Since we're going to shut down the tree regardless (it has the `WT_BTREE_BULK` flag set), what kind of problem are you worried about?